src/jrd/blp.h

```diff
@@ -267,5 +267,6 @@ static inline constexpr struct
 	{"default2", default2},
 	{"current_schema", zero},
 	{NULL, NULL},	// flags - part of header
+	{NULL, NULL},
```

Comment:
This should have a comment about blr_within_group_order as normally every main code is there.